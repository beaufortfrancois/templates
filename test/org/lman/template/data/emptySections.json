{
  "list": [1,2,3],
  "block": {
    "list": [1,2,3]
  }
}
