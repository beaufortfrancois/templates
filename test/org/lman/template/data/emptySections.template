The following is a section with no content. It should not render.
{{#list}}{{/}}

Now, a section with no content within another section. It also should not render.
{{#block}}{{#list}}{{/list}}{{/block}}
