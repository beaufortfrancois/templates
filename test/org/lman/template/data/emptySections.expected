The following is a section with no content. It should not render.


Now, a section with no content within another section. It also should not render.

